tools/pyvmi/glib_build.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+
+from cffi import FFI
+import pkgconfig
+
+cdef = """
+typedef int    gint;
+typedef gint   gboolean;
+typedef void* gpointer;
+typedef const void *gconstpointer;
+typedef unsigned int    guint;
+typedef struct _GHashTable  GHashTable;
+
+typedef guint (*GHashFunc)(gconstpointer key);
+typedef gboolean (*GEqualFunc)(gconstpointer a, gconstpointer b);
+
+GHashTable* g_hash_table_new               (GHashFunc       hash_func,
+                                            GEqualFunc      key_equal_func);
+
+gboolean    g_hash_table_insert            (GHashTable     *hash_table,
+                                            gpointer        key,
+                                            gpointer        value);
+
+void        g_hash_table_destroy           (GHashTable     *hash_table);
+
+guint    g_str_hash     (gconstpointer  v);
+gboolean g_str_equal    (gconstpointer  v1, gconstpointer  v1);
+"""
+
+
+ffi = FFI()
+includes = pkgconfig.cflags('glib-2.0').replace('-I', '').split(' ')
+# set source
+ffi.set_source("_glib",
+    """
+    #include <glib.h>
+    """,
+    libraries=['glib-2.0'], include_dirs=includes)
+
+ffi.cdef(cdef)
+
+if __name__ == "__main__":
+    ffi.compile()
```

Comment:
You might want to add the `verbose=True` flag here, it helps spotting issues at compile time.

Reply:
maybe adding a command line flag `--debug` ?
Or having a verbose compilation is not a big deal ?

Reply:
Actually this will be called by the `setup.py`, so it's better to have a verbose output always enabled.